src/code/client.js

```diff
@@ -1132,14 +1134,25 @@ class CloudFileManagerClient {
   }
 
   _setWindowTitle(name) {
-    if (!this.appOptions.appSetsWindowTitle && __guard__(this.appOptions != null ? this.appOptions.ui : undefined, x => x.windowTitleSuffix)) {
-      return document.title = `${(name != null ? name.length : undefined) > 0 ? name : (tr("~MENUBAR.UNTITLED_DOCUMENT"))}${this.appOptions.ui.windowTitleSeparator}${this.appOptions.ui.windowTitleSuffix}`
+    if (this.appOptions?.appSetsWindowTitle) {
+      return
+    }
+    const {ui } = this.appOptions
+    if(ui) {
+      const { windowTitleSeparator, windowTitleSuffix } = ui
+      if (windowTitleSuffix) {
+          const displayName = (name || "").length > 0
+            ? name
+            : tr("~MENUBAR.UNTITLED_DOCUMENT")
+          document.title = `${displayName}${windowTitleSeparator}${windowTitleSuffix}`
+      }
     }
   }
 
   _getHashParams(metadata) {
-    let openSavedParams
-    if (__guard__(metadata != null ? metadata.provider : undefined, x => x.canOpenSaved()) && ((openSavedParams = __guard__(metadata != null ? metadata.provider : undefined, x1 => x1.getOpenSavedParams(metadata))) != null)) {
+    let openSavedParams = metadata?.provider?.getOpenSavedParams(metadata) || null
```

Comment:
I think you can get rid of the `|| null` as `metadata?.provider?.getOpenSavedParams(metadata)` will return either the params or undefined.